scss/components/_award.scss

```diff
@@ -5,6 +5,10 @@
 }
 
 .civiawards__award__basic-details {
+  label {
+    width: 200px;
```

Comment:
is there an existing variable we can use here?

Reply:
No there is no such variable. At the moment, I dont think this will be reused so moving to variables does not make sense I feel.